src/components/stopPoster/stopPoster.js

```diff
@@ -31,8 +32,12 @@ class StopPoster extends Component {
         super(props);
 
         this.onError = this.onError.bind(this);
+        this.setMapHeight = this.setMapHeight.bind(this);
+        this.setRightColumnHeight = this.setRightColumnHeight.bind(this);
 
         this.state = {
+            rightColumnHeight: -1,
+            mapHeight: -1,
```

Comment:
We need to know both the height of the map as well as the height of the column that houses the map and the route diagram.